src/lib/app-switcher/app-switcher-menu/app-switcher-menu.component.ts

```diff
@@ -13,17 +15,21 @@ export class VantageAppSwitcherMenuComponent {
   @Input() otherProducts: IVantageAppSwitcherItem[];
   @Input() exploreMoreLink: string;
 
-  constructor(private elRef: ElementRef) {}
+  @ViewChild('expansionPanel') expansionPanel: MatExpansionPanel;
+
+  constructor(private elRef: ElementRef, private _changeDetectorRef: ChangeDetectorRef) {}
 
   _blockEvent(event: Event): void {
     event.preventDefault();
     event.stopPropagation();
   }
 
   scrollToBottom(): void {
+    this._changeDetectorRef.detectChanges();
     setTimeout(() => {
       const elem: HTMLElement = this.elRef.nativeElement.querySelector('.td-menu-content');
       elem.scrollTop = elem.scrollHeight;
+      this._changeDetectorRef.detectChanges();
```

Comment:
Does this not work with `markForChange`? 

detectChanges means we did something wrong if we have to use it for the most part.